packages/react/index.js

```diff
@@ -72,6 +72,7 @@ export {
   createFactory,
   useTransition,
   useTransition as unstable_useTransition,
+  startTransition as unstable_startTransition,
```

Comment:
Does this also need an unprefixed export like others?

Reply:
Nice catch, it doesn't need it but yes it should have it.